CONTRIBUTING.rst

```diff
@@ -97,15 +97,18 @@ Ready to contribute? Here's how to set up `orsopy` for local development.
 Pull Request Guidelines
 -----------------------
 
-Before you submit a pull request, check that it meets these guidelines:
+Before you submit a pull request of your feature branch against the `main` branch of the `orsopy` repository,
+check that it meets these guidelines:
 
-1. The pull request should include tests.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 3.5, 3.6, 3.7 and 3.8, and for PyPy. Check
-   https://travis-ci.com/reflectivity/orsopy/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+1. The pull request should include tests for the new functionality. Run the tests in your local machine with `pytest`.
```

Comment:
This list won't be ordered now. 2 appears to come before 1.

Reply:
@aglavic this list is unordered now, and the formatting is wrong. It would've been better to fix the PR as asked.

Reply:
@andyfaff I have a mind of creating a conda recipe for orsopy so I will include fixes for all your comments in my next PR 😁

Reply:
There's a staged recipe on condaforge that needs the cf maintainers to do something

Reply:
Sorry, I thought those would have been done already. My fault.